kubernetes/samples/scripts/domain-lifecycle/startCluster.sh

```diff
@@ -0,0 +1,100 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+
+function usage() {
+
+  cat << EOF
+
+  This script starts a WebLogic cluster in a domain by patching it's
+  'serverStartPolicy' field to 'IF_NEEDED'. This change will cause the
```

Comment:
consider updating the 'serverStartPolicy' reference here (and in similar usage doc) so that its fully qualifie.  'spec.xxx.SERVER-NAME.xxx.serverStartPolicy'...

Reply:
Fixed